src/cvl/custom_validation/sonic_sflow.go

```diff
@@ -0,0 +1,31 @@
+package custom_validation
+
+import (
+	"net"
+	util "cvl/internal/util"
+ )
+
+
+//Validate ip address of sflow collector ip 
+func (t *CustomValidation) ValidateIp(vc *CustValidationCtxt) CVLErrorInfo {
+	ip := net.ParseIP(vc.YNodeVal)
+	if ip == nil {
+	         util.CVL_LEVEL_LOG(0,"%s","IP IS NIL")
+                 return CVLErrorInfo{
+			            ErrCode: CVL_SYNTAX_INVALID_INPUT_DATA,
+				    TableName: "SFLOW_COLLECTOR",
+				    CVLErrDetails : "IP address is not valid",
```

Comment:
Please populate 'ConstraintErrMsg' field with error message like "Sflow Collector IP is not valid", REST server uses this field to throw the error message on CLI console.

Reply:
Done

Reply:
Did you miss this one ?